.gitignore

```diff
@@ -6,6 +6,8 @@
 /pkg/
 /spec/reports/
 /tmp/
+Gemfile.lock
```

Comment:
Can you elaborate on this decision? If we don't have lock, our tests may not be stable in the future (not all dependencies have pinned versions)

Reply:
https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/
> Do not check your Gemfile.lock into version control

Reply:
Though it might be outdated. Will take a closer look tomorrow.
https://github.com/rubygems/rubygems/issues/3372

Reply:
Yeah bundler recommends checking it in.
https://bundler.io/v2.5/man/bundle-install.1.html#THE-GEMFILE-LOCK
> As a result, you SHOULD check your Gemfile.lock into version control, in both applications and gems.